docker/cygnus-ngsi/Dockerfile

```diff
@@ -16,7 +16,7 @@
 # For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
 #
 
-FROM debian:11.6-slim
+FROM debian:12.1-slim
```

Comment:
Should this change be propagated to the rest of the family of Dockerfiles we have in this repo?

Reply:
Or maybe not :)... it seems they are still using old CentOS images (issue https://github.com/telefonicaid/fiware-cygnus/issues/2285)